helm-chart/templates/spire-server.yaml

```diff
@@ -214,10 +214,17 @@ spec:
     spec:
       serviceAccountName: spire-server
       shareProcessNamespace: true
+      securityContext:
+        fsGroup: 1000 #spire
+        runAsNonRoot: true
+        runAsUser: 1000 #spire
       containers:
       - name: spire-server
-        image: {{ include "spire.image-server" . }}/spire-server:1.5.4
+        image: {{ include "spire.image-server" . }}/spire-server:1.6.1
         imagePullPolicy: {{ .Values.registry.imagePullPolicy }}
+        securityContext:
+          allowPrivilegeEscalation: false
+          privileged: false
```

Comment:
does the security context in the parent `spec` apply to all containers? If so, could these just live there instead of in each container?

Reply:
If I move these up to the parent spec I get the messages:

> `W0406 08:14:29.767383   93141 warnings.go:70] unknown field "spec.template.spec.securityContext.allowPrivilegeEscalation"

> W0406 08:14:29.767469   93141 warnings.go:70] unknown field "spec.template.spec.securityContext.privileged"`

I think these fields are not available in a security context at the parent spec level, only at the container.